### PR TITLE
Clear data when time bounds are changed

### LINKED ISSUE #5456: Bar Graph does not update when time conductor value changes
#### Summary
Changing time conductor in fixed or local clock mode does not update the graph with new data.

#### Expected vs Current Behavior
Bar Graph should be updated to show relevant data to the time conductor.

#### Steps to Reproduce
1. Drag the analog aggregate to a bar graph
2. Expect that some bars are displayed (this is currently not happening)
3. Change time conductor start/end
4. Expect that some bars are displayed with new data.

#### Environment

* Open MCT Version: release/2.0.5
* Deployment Type: VIPER testathon
* OS: all
* Browser: all

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by shefalijoshi**
Closes #5456 

### Describe your changes:
Ensure that the yKey is set to 'none' if there is no range with array Values
React to clear data action

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/charts/bar/BarGraphView.vue [hunk 1]
```diff
@@ -236,7 +236,32 @@ export default {
         refreshData(bounds, isTick) {
             if (!isTick) {
                 const telemetryObjects = Object.values(this.telemetryObjects);
-                telemetryObjects.forEach(this.requestDataFor);
+                telemetryObjects.forEach((telemetryObject) => {
+                    //clear existing data
+                    const key = this.openmct.objects.makeKeyString(telemetryObject.identifier);
+                    const axisMetadata = this.getAxisMetadata(telemetryObject);
+                    let trace = {
```
Comment (scottbell) [inline src/plugins/charts/bar/BarGraphView.vue:243]: This trace looks like it's copied/pasted from below. Can we generalize this into a method?

File: src/plugins/charts/bar/inspector/BarGraphOptions.vue [hunk 2]
```diff
@@ -316,11 +316,16 @@ export default {
                     }
                 } else {
                     if (this.yKey === undefined) {
-                        yKeyOptionIndex = this.yKeyOptions.findIndex((option, index) => index !== xKeyOptionIndex);
-                        if (yKeyOptionIndex > -1) {
+                        if (metadataValues.length && metadataArrayValues.length === 0) {
                             update = true;
-                            this.yKey = this.yKeyOptions[yKeyOptionIndex].value;
-                            this.yKeyLabel = this.yKeyOptions[yKeyOptionIndex].name;
+                            this.yKey = 'none';
```
Comment (scottbell) [inline src/plugins/charts/bar/inspector/BarGraphOptions.vue:321]: not following this code exactly, but it looks like we're setting the options to `none` if there aren't any options in the metadata?

**Review by scottbell — CHANGES_REQUESTED** [review]
Small change, and small question 😀
good work!

File: src/plugins/charts/bar/inspector/BarGraphOptions.vue (hunk 2 above)
Comment (shefalijoshi) [reply src/plugins/charts/bar/inspector/BarGraphOptions.vue:321]: That's correct.

**Review by scottbell — APPROVED** [review]
LGTM!

**Pull request merged**